packages/server/src/ChunkExtractor.js

```diff
@@ -484,6 +487,32 @@ class ChunkExtractor {
     const assets = this.getPreAssets()
     return assets.map(asset => assetToLinkElement(asset, extraProps))
   }
+
+  getLinkTagsSince(extraProps = {}) {
+    const assets = this.getPreAssets()
+    const linkTags = assets.map(asset => {
+      if (!this.seenChunks.includes(asset.chunk)) {
+        this.seenChunks.push(asset.chunk)
+        return assetToLinkTag(asset, extraProps)
+      }
+      return false
+    })
+    return joinTags(linkTags.filter((tag)=>Boolean(tag)))
+  }
+
+  getScriptTagsSince(extraProps = {}) {
```

Comment:
Using `since` creates a question `since what`?
It's easier to use a _desctructive_ naming pattern:
- drainScriptTags
- flushScriptTags